### DISCUSSION TITLE
Crowdsourcing: Common issues with NCA datasets

### DISCUSSION THREAD

**Discussion by billdenney** [humanpred/pknca#327] [Ideas]
We are starting a project where PKNCA will get a large bolus of work to make it both easier to use and more efficient to automate (among other goals). With that, one item that comes up often are how can outliers that may be potential issues or errors in data be identified. Please share any thoughts you have that could make data issues easier with PKNCA!

**Comment by Gero1999** [comment]
I know this message is old, so I am not sure if this is still a concern. But based on what I found at discussions, if I had to give an answer, I would bet for:

- Make more visible on the website / README how to report bugs. Including a specific link for issues and another one for feat-requests 
- Creating issue or discussion-bug templates so users know better how to report problems (when possible with reproducible examples)
- Ask openly on social media for critical and honest feedback. This might attract non-convinced users to share their experience on why they are not using the package
